hack/setup-bob-4.4.sh

```diff
@@ -0,0 +1,39 @@
+if [[ -z "$KUBECONFIG" ]]; then
```

Comment:
@alaypatel07 how would you feel about renaming this script to something like `setup-migtenant.sh`

Reply:
sure, good idea, may be `setup-users-4.4.sh`? the script is not actually creating a MigrationTenant resource so wanted to avoid that confusion

Reply:
@alaypatel07 I think we should mention in the script name that this is related to non-admin. Also, it looks to me like we are creating a MigTenant in this script.